docs/design_decisions.md

```diff
@@ -1,203 +1,245 @@
+
+### DD5.2 We maken hergebruik van yaml-componenten d.m.v. absolute links
+Bij hergebruik van Yaml-componenten wordt gebruik gemaakt van absolute links zodat de openapi.yaml in alle editors onderhoudbaar is en dus iedereen die een bijdrage wil leveren een pull-request kan indienen. 
+
+_**Ratio**_
+
+We willen bijdragers tijdens het ontwikkeltraject niet dwingen om de file-structuuur van de provider-developer over te nemen. Dit leidt tot onnodige complexiteit op het ontkoppelpunt van provider en consumer tijdens het ontwikkeltraject. 
+
+_**Kanttekening**_
+
+Nu verwijzen de links nog naar de master-branch van Haal-centraal-common. Op het moment dat de specificaties van 1 van de API's definitief wordt zal ook de common geversioneerd moeten worden (d.m.v. een release op github) 
+
+### DD5.3 Het toepassen van Patterns is beperkt tot parameters
+
+Bij een specifieke bevragingen-API is het toepassen van Patterns beperkt tot de parameters.  
+
+### DD5.4 oneOf constructies worden niet gebruikt in de API-sepcificaties
+Alhoewel de oneOf constructie een valide OAS3 constructie is levert deze bij het genereren van code problemen op.
+Deze constructie wordt ontweken door twee mogelijke alternatieven. 
+* De subtypes worden samengevoegd tot 1 object en er wordt een typeveld toegevoegd om te duiden welk subtype het betreft. Deze keuze is logisch als de subtypes grotendeels overlappen.
+* De subtypes worden volledig opgenomen als property van een object. In het response is altijd maar 1 van deze properties gevuld. Deze keuze is logisch als de subtypes weing gemeenschappelijke properties hebben.
+
+_**Ratio**_
+
+Diverse code-generatoren gaan dus niet goed om met deze constructie en genereren foutieve code.
+
+_**Kanttekening**_
+
+Mochten code-generatoren in de toekomst wel goed met deze constructie om kunnen gaan dan is het het overwegen waard om deze constructie aan te passen bij de eerstvolgende major (breaking) change.
+
+### DD5.5 Gemeentelijke kerngegevens en plusgegevens worden niet opgenomen in de resource
+In de response worden alleen gegevens opgenomen die in de basisregistratie worden vastgelegd. 
+
+_**Ratio**_
+
+Deze gegevens worden niet vastgelegd in een (voor alle gemeenten geldend) bronsysteem dat voor de bevraging geraadpleegd kan worden. Deze gegevens zijn dus (voorlopig) niet raadpleegbaar. Ook worden deze gegevens niet in alle gemeenten (op dezelfde manier) gebruikt.
+
+### DD5.6 De API filtert terug te geven gegevens op autorisatie van de organisatie
```

Comment:
is het niet beter een document te maken met ontwerprichtlijnen voor API ontwerp, gericht op hoe de API specificaties eruit moeten zien, en een ander document met dit soort architectuurprincipes?